.vscode/launch.json

```diff
@@ -279,6 +279,7 @@
 			"request": "launch",
 			"name": "Main Process",
 			"attachSimplePort": 5875,
+			"enableContentValidation": false,
```

Comment:
This is needed since we modify the .js file (to add the sourceURL) so contents don't line up. I will make a followup change in js-debug to not validate content on files with `hasSourceURL: true` since I think those will _never_ match, similar to sourcemapped content.